<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunGame</class>
 <widget class="QDialog" name="RunGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border-image: url(:/Background/Resources/RunGame/Background/default.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressBar_time">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    color : solid gray;
    border: 2px solid gray;
    border-radius: 5px;
    padding: 0px;
    text-align : center ;
}
QProgressBar::chunk {
        background-color:#CD96CD;
        width:10px;
        margin:0.5px;
}</string>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_time_number">
         <property name="styleSheet">
          <string notr="true">QLabel {
        font: 12pt &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>60</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_pause">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_normal.png);
	font: 75 18pt &quot;华文彩云&quot;;
}
QPushButton:hover {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_hover.png);
    color: rgb(240, 20, 30);
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    color: rgb(150, 10, 20);
}
</string>
         </property>
         <property name="text">
          <string>暂停游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_continue">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_normal.png);
	font: 75 18pt &quot;华文彩云&quot;;
}
QPushButton:hover {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_hover.png);
    color: rgb(240, 20, 30);
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    color: rgb(150, 10, 20);
}
</string>
         </property>
         <property name="text">
          <string>继续游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rearrange">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_normal.png);
	font: 75 18pt &quot;华文彩云&quot;;
}
QPushButton:hover {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_hover.png);
    color: rgb(240, 20, 30);
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    color: rgb(150, 10, 20);
}
</string>
         </property>
         <property name="text">
          <string>重新排列</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_back">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_normal.png);
	font: 75 18pt &quot;华文彩云&quot;;
}
QPushButton:hover {
	border-image: url(:/Button/Resources/RunGame/Button/btn_menu_hover.png);
    color: rgb(240, 20, 30);
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    color: rgb(150, 10, 20);
}
</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
